scripts/config.pl

```diff
@@ -102,6 +103,7 @@
 MBEDTLS_NO_64BIT_MULTIPLICATION
 MBEDTLS_PSA_CRYPTO_SPM
 MBEDTLS_PSA_INJECT_ENTROPY
+MBEDTLS_ECP_RESTARTABLE
```

Comment:
Previoulsy, the only builds that were tested with `ECP_RESTARTABLE` enabled were builds with full configs. Now that `config.pl full` no longer enables it, we probably want a new component in `all.sh` that builds with `USE_PSA_CRYPTO` disabled and `ECP_RESTARTABLE` enabled, and then runs the tests suites, including `ssl-opt.sh -f 'EC restart'`.

Reply:
Can this be handled separately, after merging this PR, or do you consider it a blocker?

Reply:
Given that we may want to always set `USE_PSA_CRYPTO` shortly, it seems a bit of a longer term activity to get ECP_RESTARTABLE working with it.

Reply:
I don't consider this a blocker, but it postponed, it needs to be tracked in order not to be forgotten - do you already have a ticket open for that?

Reply:
Defect IOTSSL-2848 has been created for that.